CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ FEATURES:
 * `r/tfe_team`: Add attribute `manage_membership` to `organization_access` on `tfe_team` by @JarrettSpiker ([#801](https://github.com/hashicorp/terraform-provider-tfe/pull/801))
 
 ENHANCEMENTS:
+* Remove beta flag from `r/tfe_project_variable_set` test. Feature now generally available.
```

Comment:
This does not need to be listed in the CHANGELOG.md since it only affects the test file and does not add any main functionality.

Reply:
Happy to ✅  once this is removed @rberecka 💅